ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.view.js

```diff
@@ -13,8 +13,8 @@
  *
  **/
 /*global require*/
-import React from 'react'
-import CopyCoordinates from '../../react-component/presentation/copy-coordinates'
+import * as React from 'react'
```

Comment:
This isn't a typescript file so you don't need to import it like that.

```suggestion
import React from 'react'
```

Reply:
Isn't it less confusing to use the same syntax everywhere you can? It does work.